core/src/br/cefetmg/games/minigames/NinjaTooth.java

```diff
@@ -139,33 +139,35 @@ private void killTartarus(Bullet b,Tartarus enemy) {
     @Override
     public void onUpdate(float dt) {
         // atualiza os inimigos (quadro de animação + colisão com dentes)
-        if (superTooth.getHeadPosition().x > super.screen.viewport.getWorldWidth()
-                || superTooth.getHeadPosition().x < 0
-                || superTooth.getHeadPosition().y > super.screen.viewport.getWorldHeight()
-                || superTooth.getHeadPosition().y < 0) {
-
-            superTooth.wasHurt();
-            super.challengeFailed();
-            toothBreakingSound.play();
-        }
+        if (!isPaused) {
```

Comment:
Evite criar um pull request que altere outras coisas que não aquela para o qual ele foi criado.

Além disso, esta modificação é necessária pra que o pause funcione? Se for assim, será necessário modificar todos os MiniGames e isso seria péssimo (manutenibilidade 0).